src/com/pentalog/pentastagiu/homework2/Account.java

```diff
@@ -0,0 +1,35 @@
+package com.pentalog.pentastagiu.homework2;
+
+import javax.xml.transform.stream.StreamSource;
+
+public class Account {
+    private int amount;
+    private String currency;
+    private String name;
+
+    public Account(int amount, String currency, String name) {
+        this.amount = amount;
```

Comment:
ar trebui adaugata si o validare pe amount, de vreme ce implementam in withdraw logica de a nu permite  o balanta negativa, ar trebui sa avem grija si aici